server/routes/createGoal/createGoalController.ts

```diff
@@ -77,6 +107,21 @@ export default class CreateGoalController {
     return res.redirect(`/plan/${prisonNumber}/goals/${goalIndex}/add-step/1`)
   }
 
+  // TODO: RR-748 - Implement submit handler for new create goal journey
+  newSubmitCreateGoalForm: RequestHandler = async (req, res, next): Promise<void> => {
+    const { prisonNumber, goalIndex } = req.params
+
+    req.session.newGoal.createGoalForm = { ...req.body }
```

Comment:
As above, this assumes we will be using the existing form object and validating it in the same way. I'm really not sure that will be the case .... 🤔 

For the time being, can you remove all this stuff, and just make this submit handler method return the res.redirect